### PR TITLE
Items wrap long text

### PR DESCRIPTION
**Pull request by farskid**
Added styles to wrap very long texts.

Now my todometer looks like below on long-text items:

longtext

### CODE REVIEW

File: styles/item.less [hunk 1]
```diff
@@ -3,19 +3,20 @@
...
   box-sizing: border-box;
   font-size: @item-font-size;
   .item-name {
     width: 400px;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    overflow: hidden;
+    word-break: break-all;
+    &:first-letter {
+      text-transform: uppercase;
```
Comment (cassidoo) [inline styles/item.less]: I'd remove this section, the first letter doesn't need to be uppercase (you can see in your screenshot that the node modules path started with an uppercase, which wasn't needed)

File: styles/item.less [hunk 2]
```diff
@@ -3,19 +3,20 @@
   align-items: center;
   justify-content: space-between;
   margin: 0 0 10px 0;
-  padding: 0 20px;
+  padding: 10px 20px;
   width: 100%;
-  height: 70px;
+  // height: 70px;
```
Comment (cassidoo) [inline styles/item.less]: Make this `min-height` instead of leaving a comment in there

File: styles/item.less [hunk 3]
```diff
@@ -3,19 +3,20 @@
   align-items: center;
   justify-content: space-between;
   margin: 0 0 10px 0;
-  padding: 0 20px;
+  padding: 10px 20px;
```
Comment (cassidoo) [inline styles/item.less]: Is this necessary?

Comment (farskid) [inline styles/item.less]: For the long text being wrapper, it gets glued tied to the borders. the 10px is there to make a distinction between the text and the borders. what do you think?

File: styles/item.less (hunk 2 above)
Comment (farskid) [inline styles/item.less]: That I'm gonna fix.

File: styles/item.less (hunk 1 above)
Comment (farskid) [inline styles/item.less]: You're right. Thought that also would be a good looking, but it violates the `Single duty PR` rule 👍

File: styles/item.less (hunk 3 above)
Comment (cassidoo) [inline styles/item.less]: Ah, I see.  Let's just make this `padding: 20px` so it's 20px all around. :)

Comment (farskid) [inline styles/item.less]: You got it. 👍

**Comment by farskid** [comment]
Updated PR with requested changes.

**Comment by cassidoo** [comment]
Thanks for doing this! :)

**Comment by farskid** [comment]
Always a pleasure :)

**Pull request merged**